Dockerfile

```diff
@@ -4,7 +4,18 @@ MAINTAINER 583114@bah.com
 VOLUME /root/.m2
 
 WORKDIR /home
-COPY . .
+
```

Comment:
Looks like this file may have gone the wrong direction. We want the `COPY . .` now that we have the `VOLUME` definition.

Reply:
We don't want to copy the entire current folder to the image. If you have garbage and large files, docker runs out of space.